builtin/staker/staker.go

```diff
@@ -539,11 +532,18 @@ func (s *Staker) WithdrawDelegation(
 		return 0, err
 	}
 
+	if del == nil {
+		return 0, NewReverts("delegation is empty")
+	}
+
 	val, err := s.validationService.GetValidation(del.Validation)
 	if err != nil {
 		return 0, err
 	}
 
+	if val == nil {
```

Comment:
Same as `SignalDelegationExit` should match `// there can never be a delegation pointing to a non-existent validation`?